src/config/README.md

```diff
@@ -0,0 +1,68 @@
+## Usage (in Python)
+
+
+### Config JSON file
+Can set config data in a JSON file that is passed to the config.load() as the first parameter.
```

Comment:
Since this is documentation, I think this should go in docs/ .

Reply:
I'll move it